client/app/bundles/HelloWorld/components/ResturantItem.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { PropTypes } from 'react';
+
+export default class ResturantItem extends React.Component {
+    componentDidMount(){
+        console.log("mounted");
+        console.log(this.props);
+    };
+
+    constructor (props) {
+        super(props);
+        this.state = {
+            resturant: JSON.stringify(this.props.resturant),
+            average_rating: this.props.average_rating,
+            cuisine_image: this.props.cuisine_image,
+            cuisine_title: this.props.cuisine_title,
+            tenbis_image: this.props.tenbis_image
```

Comment:
I would avoid putting these in the state unless they need to change locally, meaning for example some action coming from this component changes those values inside itself.

For most occasions `props` would do just fine.

Also, the general practice is `camelCase`-named props, and not `snake_case`-named. :)